core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -143,13 +146,6 @@ public void removeNetworks(String identifier) {
 
     private void removeNetwork(String networkName) {
         try {
-            try {
-                // First try to remove by name
-                dockerClient.removeNetworkCmd(networkName).exec();
```

Comment:
@rnorth ouch. It doesn't fail when recently added Docker Networks support is being used (because we know the IDs), so I wouldn't remove it.

Since we list networks in Docker Compose's implementation, maybe we can use the IDs there as well?